PluginInstaller/Create-PluginInstaller.ps1

```diff
@@ -28,8 +28,8 @@ param(
   #$lib = "$wix\\WixUI.wixlib",
   $ui_ref = "WixUI_Mondo",
   $title = "UserPlugins",
-  $manufacturer = "Citrix",
-  $description = "XenCenter Plugins",
+  $manufacturer = $env:MANUFACTURER_NAME,
+  $description = $env:PRODUCT_NAME + " Plugins",
   $product_version = "1.0.0.0",
   $upgrade_code = "8282b90a-cb51-4c02-a1c1-ecfcff9861bf",
   $product_code = "8282b90a-cb51-4c02-a1c1-ecfcff9861bf",
```

Comment:
Understood.
Let's merge this and create another ticket on the backlog to tidy things up.